switchlink/switchlink_link.c

```diff
@@ -205,7 +296,25 @@ void switchlink_process_link_msg(const struct nlmsghdr *nlmsg, int msgtype) {
   if (msgtype == RTM_NEWLINK) {
     switch (link_type) {
       case SWITCHLINK_LINK_TYPE_BRIDGE:
+        break;
+      case SWITCHLINK_LINK_TYPE_TEAM:
+        krnlmon_log_debug("LAG via teaming driver isn't supported\n");
```

Comment:
isnt it better to use krnlmon_log_info ?

Reply:
Addressed.